LICENCE.txt

```diff
@@ -1,11 +1,30 @@
+"Source code for Antinet / Galaxy / Yedino - Licence"
 
 The code and files in this directory (but with exception of code from other
 sources, read the code carefully and look for more LICENCE/LICENSE files)
-are now Copyrighted (C) 2015-2016
-by the "Antinet.org team",
-they are licensed hereby as GPL version 3,
-and at discretion of the authors it may be licensed on more liberal license.
+are now Copyrighted (C) 2015-2016 by the Yedino,
+they are licensed hereby as GPL version 3 with following exception:
+this code at discretion of the Yedino may be freely relicensed (including
+e.g. adding new licence(s) or removing current licence(s)) as described below.
```

Comment:
We may not remove current license, it is forbidden in the GPL3 terms. Only add more licenses.